src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, Countable {
+    private String color;
+    private String name;
+
+    public Figure() {

```

Comment:
An empty default constructor is redundant when there are no other constructors. However, in this case, since there are overloaded constructors, this default constructor can be kept if it's required for instantiation without parameters.